firebase.enc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "pipelinedashboard-khush",
+  "private_key_id": "c5d9b38b46b41959a7e1c41fdcaeedb5251a9f64",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDSeJ04EWHH9HEW\ngAKpngq8sXGExvrL7Lka/xS4LhZC+IXPnK86PYNUrJ7X8vfNrgq74TKbC0wSzgdi\nGZuQXY3rmB5HnKxY9xHivED110hkVpMjcZy4tfR2EMrLiTS6s2pBhK7kjcg+SbVb\nbxZylqJrmF81vLnON8ky0KGdJG90zIee7kJDBx7xQDkDnIF5sNz+0IV5fUsjXjWx\nrmzigrNELsjAmvB274jQo8GaPxJaUNLsATxwdd7PgRJq2l6fH9EkzI1lizQAKXqx\ns9JRiOLMpvWj5RBF91g3XURsnyxVpF5WJc3fCdld4Ew0cEzQclYJfZjcPUQLdAUW\nnGfTr/pdAgMBAAECggEACpJ8c0CMRNsGgzbS62pA59GR/jVwi6XMdoqxo77M53hM\nYG1ib+5/FlXFdNuDjxVa/K0HObqLi7ZFsKvQoKPSaS8u9jHshluCznzeVnsF6+Rd\nZ75hPPYG3ZdiWG7/8xDoGXpsvpZQpLBXJ8yguAjRB86TmcNtIeqzdJX+SEJ4RsmY\nAVbdp5I5ySv2L9IlDuDTNsUpnAe5N8nVtLDbmSH5z5ksJ0spfg6tUbnyT0tH99kR\nJGo908wrkxarm3Aoujv7BZXdDyOzGWieL9dsl50sk464/qooF+TaU2XtKcoe17Mt\nKjTBs2nDBeB47hkjdBPHSd09PbhV7uLW/cSK1s0ikQKBgQDv0kqRfzK54HWuVI57\naw0sc/6valOMHhjF27PqezdRosk20KUECqydvK/aWeioi0F/UeIXKbh1zwe9frGx\n7ZxLxYSYXpSNyIPTbR3lyXRNfeuqR7K1bBlCpIOub25docwDfEEH5T6eym7Wx5Lv\nimrpKuj3bEaJmd4mHVuLmZ6jkwKBgQDgq3GwQ672YEkxHcPseJA9yy3hKh8Su5/W\nmW8iIc5rJpu2mxhjxiTLUZliBbwvtXkCTQcFFDF7uRE6oTZZJNxiuRuU1SKlMrEx\npxDB/dVsgI4l7Od8mMhy3woRMJIegKy9WUcTGs7FpYYUzDLIzgVMFKnaIXRsHil6\nJNZueYaATwKBgBVQyqHt/VM6GRualmNv22blFHEA8VYV4rvLUE16ECuoXVwK5W/b\nhwYqHDM7JIu+fN7VJ1RJuAR4l0aHeUHITckKUEO+vFNxzA3BuBOReU3d8FFdOAwa\nBJDA0JdzfgrCc8CnjZwnCw7QDVP3vx6dXe1FjBlZvVdDntQibd2VlN0TAoGAAosu\nYFaD1ppEed/iZfz/dROl01TMwUOKjb1y+qnzUlc/kBO1dPu7Uun7mJaECO4/Dn2c\nxLvzy3jO7yEAIoY+UIfT/h1jpQcr5moD0I51KiNSeK9z938rVdGcpZEtNlcynXoz\nSc4V5Tr4h6fAM3HRsYjSW3vhe4/j3FQqcVnC4hUCgYBNUJAwnNmbXzhuPxf6GytU\nti0SeX40n3YoWOU4ShvLAn9zZqmJSeTqlOV8Iuofia8e9ixXit3M7xmRYwPoezms\nAE5ZKA/czS9b6xCZ/kNrMc/jNxYWw5fZA3MSmPZH+GeLInRIK0r0ufAVq6aHU5r2\nAIu2esw5TrtdsDm2G6n4KA==\n-----END PRIVATE KEY-----\n",
+  "client_email": "firebase-adminsdk-71v0k@pipelinedashboard-khush.iam.gserviceaccount.com",
+  "client_id": "114902040977932187098",
+  "auth_uri": "https://accounts.google.com/o/oauth2/auth",
+  "token_uri": "https://oauth2.googleapis.com/token",
+  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
+  "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/firebase-adminsdk-71v0k%40pipelinedashboard-khush.iam.gserviceaccount.com"
+}
```

Comment:
This file should never be committed - it is part of the `gitignore`

Reply:
I have now reset all keys for your environment

Reply:
okay

Reply:
This file needs to be removed from the PR

Reply:
I will remove